falcon/response.py

```diff
@@ -279,7 +279,7 @@ def set_stream(self, stream, content_length):
         self._headers['content-length'] = str(content_length)
 
     def set_cookie(self, name, value, expires=None, max_age=None,
-                   domain=None, path=None, secure=None, http_only=True):
+                   domain=None, path=None, secure=None, http_only=True, same_site=None):
```

Comment:
Please document the `same_site` parameter in the docstring. Without that, it's not clear what type it expects.

Reply:
Yeah, a PR isn't complete, will do.

Reply:
As @RonRothman points out, let's add documentation for public methods.
I'm reiterating the obvious here but I do so that these stay as requested changes.

It would also be cool to link to the relevant RFC section as docs for other parameters do.